ligo/src/types.mligo

```diff
@@ -96,6 +96,7 @@ type proposal =
   ; metadata : proposal_metadata
   ; proposer : address
   ; proposer_frozen_token : nat
+  ; proposer_fixed_fee_in_token : nat
```

Comment:
Ah I didn't get why this was needed at first, but I get it now.

Changing the fixed fee only applies to proposals created after it, so we need to track this.
Can you add a quick comment here to note this, so we don't forget?